src/sharding/ShardClientUtil.js

```diff
@@ -199,13 +199,14 @@ class ShardClientUtil {
    */
   _respond(type, message) {
     this.send(message).catch(err => {
-      err.message = `Error when sending ${type} response to master process: ${err.message}`;
+      let error = {err}
+      error.message = `Error when sending ${type} response to master process: ${err.message}`;
```

Comment:
```suggestion
      let error = new Error(`Error when sending ${type} response to master process: ${err.message}`);
```
Create a new instance of Error with the new message.

Reply:
👍🏻 good, now people will know the error, when some happens